src/main/java/com/example/sortener/exceptions/ErrorResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.example.sortener.exceptions;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ErrorResponse {
+    public ErrorResponse(String message, List<String> details) {
+        super();
```

Comment:
This object is not extending any class, so this super() is calling constuctor from Object.class, which doesn't do anything